KsApi/px/PerimeterXClient.swift

```diff
@@ -44,7 +44,13 @@ public class PerimeterXClient: NSObject, PerimeterXClientType {
       [.Block, .Captcha].contains(response.type)
     else { return false }
 
-    NotificationCenter.default.post(name: Notification.Name.ksr_perimeterXCaptcha, object: response)
+    DispatchQueue.main.async {
+      NotificationCenter.default.post(
+        name: Notification.Name.ksr_perimeterXCaptcha,
+        object: nil,
+        userInfo: ["response": response]
```

Comment:
This wasn't correct before, according to the [docs](https://developer.apple.com/documentation/foundation/nsnotificationcenter/1410608-postnotificationname) the `object` param is meant to represent the _sending_ object and the `userInfo` param is meant to contain additional data to carry along with the notification which is what we're really doing here.

Reply:
ah wow, this one definitely slipped through the cracks while I reviewed as well. good catch.